Update VCR cassettes

Cruikshanks opened the pull request:
Every couple of weeks we update our VCR cassettes to ensure our mock requests match the current real world ones.

Cruikshanks committed:
Update VCR cassettes

Every couple of weeks we update our VCR cassettes to ensure our mock requests match the current real world ones.

This also lead to some other dependency updates when run locally hence the Gemfile.lock update.

Cruikshanks committed:
Fix rubocop error

...by deleting the offending line!

This was for when we started a new gem by first creating it within a namespace within this project. That gem eventually became defra-ruby-exporters, which was replaced by defra-ruby-aws. But long and short is that it don't live here no more!

Cruikshanks commented:
Codeclimate is just being it's usual self. Should wake up once this is merged.

cintamani commented on Gemfile.lock:
```diff
@@ -157,7 +157,7 @@ GEM
       rails (>= 4.1.0)
       sass (>= 3.2.0)
-    govuk_frontend_toolkit (8.1.0)
+    govuk_frontend_toolkit (9.0.0)
```
I am a bit concerned about this update, are we sure we want to apply this without a ticket to QA visual regressions?

Cruikshanks committed:
Reset Gemfile.lock changes

Resetting (and hoping it still builds) until I have 5 secs to figure out why running `bundle install` locally caused a bunch of stuff to get updated.
